README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
-conversion.
-
-See
-[Migrating from PostgreSQL to Cloud Spanner](https://cloud.google.com/spanner/docs/migrating-postgres-spanner)
-for a general discussion of PostgreSQL to Spanner migration issues.
-HarbourBridge follows most of the recommendations in that guide. The main
-difference is that we map a few more types to `STRING(MAX)`.
+Please refer below mentioned links for details on schema conversion of PostgreSQL
+and MySQL database:
+
+- [PostgreSQL schema conversion](postgres/README.md#schema-conversion)
+- [MySQL schema conversion](mysql/README.md#schema-conversion)
 
 ## Data Conversion
 
-HarbourBridge converts PostgreSQL data to Spanner data based on the Spanner
+HarbourBridge converts PostgreSQL/MySQL data to Spanner data based on the Spanner
 schema it constructs. Conversion for most data types is fairly straightforward,
 but several types deserve discussion. Note that HarbourBridge is not intended
-for databases larger than a couple of GB.
+for databases larger than a couple of GB. Please refer below mentioned links for details on
+data conversion of PostgreSQL and MySQL database:
 
-### Timestamps and Timezones
-
-As noted earlier when discussing [schema conversion of
-TIMESTAMP](#timestamp), there are some sutble differences in how timestamps are
-handled in PostgreSQL and Spanner.
-
-During data conversion, PostgreSQL `TIMESTAMPTZ` values are converted to UTC and
-stored in Spanner. The conversion proceeds as follows. If the value has a
-timezone, that timezone is respected during the conversion to UTC. If the value
-does not have a timezone, then we look for any `set timezone` statements in the
-pg_dump output and use the timezone specified. Otherwise, we use the `TZ`
-environment variable as the timezone, and failing that, we use the local system
-timezone default (as determined by Go).
-
-In constrast, conversion of PostgreSQL `TIMESTAMP` values proceeds by ignoring
-any timezone information and just treating the value as UTC and storing it in
-Spanner.
-
-### Strings, character set support and UTF-8
-
-Spanner requires that `STRING` values be UTF-8 encoded. All Spanner functions
-and operators that act on `STRING` values operate on Unicode characters rather
-than bytes. Since we map many PostgreSQL types (including `TEXT` and `CHAR`
-types) to Spanner's `STRING` type, HarbourBridge is effectively a UTF-8 based
-tool.
-
-Note that the tool itself does not do any encoding/decoding or UTF-8 checks: it
-passes through data from pg_dump to Spanner. Internally, we use Go's string
-type, which supports UTF-8.
+- [PostgreSQL data conversion](postgres/README.md#data-conversion)
+- [MySQL data conversion](mysql/README.md#data-conversion)
 
 ## Troubleshooting Guide
 
 The following steps can help diagnose common issues encountered while running
 HarbourBridge.
 
-### 1. Verify pg_dump configuration
+### 1. Verify driver configuration
+
+If you are using the dump approach then check that pg_dump/mysqldump is correctly
```

Comment:
Maybe start this paragraph with:

First, check that the driver is correctly configured to connect to your database. Driver configuration varies depending on the driver is used.

#### pgdump

If you are using pg_dump (-driver=pgdump), check that pg_dump is
correctly configured to connect to your PostgreSQL
database. Note that pg_dump uses the same options as psql to connect to your
database. See the [psql](https://www.postgresql.org/docs/9.3/app-psql.html) and
[pg_dump](https://www.postgresql.org/docs/9.3/app-pgdump.html) documentation.

Access to a PostgreSQL database is typically configured using the following
environment variables, which are standard across PostgreSQL utilities including
pg_dump:

```sh
PGHOST
PGPORT
PGUSER
```

It is also possible to configure access via pg_dump's command-line options
`--host`, `--port`, and `--username`.

#### Direct access to PostgreSQL

In this case, HarbourBridge connects directly to the PostgreSQL database
to retrieve table schema and data using the settings of the standard _PGHOST_,
_PGPORT_, _PGUSER_, _PGDATABASE_ environments variables. Password can be specified either in the _PGPASSWORD_ environment variable or provided at the password prompt.

and similarly for the other drivers.

Reply:
Done. You can check here ([LINK](https://github.com/cloudspannerecosystem/harbourbridge/blob/7ec5817781e9ecb8194e62918100a3c73a3b42f5/README.md#1-verify-driver-configuration)).